crates/test_helpers/src/lib.rs

```diff
@@ -639,43 +627,30 @@ macro_rules! test_lanes_panic {
                     core_simd::simd::LaneCount<$lanes>: core_simd::simd::SupportedLaneCount,
                 $body
 
+                // test some odd and even non-power-of-2 lengths on miri
```

Comment:
This doesn't have any even non-power-of-2. Maybe replace 5 by 6?

(Though I am also not sure why odd vs even would be an interesting difference here.)

Reply:
if you look down further it tests length 3 on miri, the idea is we want to catch bugs caused by `repr(simd, packed)` having alignment smaller than `repr(simd)` which only happens for non-power-of-2 sizes. even non-power-of-2 sizes cover where the alignment is in between the element alignment and the non-packed alignment. @calebzulawski can you add back in length 6 since that's the smallest length where that occurs?

Reply:
Yeah 3 and 6 would probably be reasonable then. To cut down on CI times I'd remove 5.